### PR TITLE
Ripgrep is also available in Ubuntu (from Cosmic)

### PR DESCRIPTION
**Pull request by sylvestre**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -302,6 +301,11 @@ If you run Debian Buster (currently Debian testing) or Debian sid, ripgrep is
 $ sudo apt-get install ripgrep
 ```
 
+From Cosmic (18.10), ripgrep is also [available in Ubuntu](https://launchpad.net/ubuntu/+source/rust-ripgrep) (same packaging as Debian).
+```
+$ sudo apt-get install ripgrep
```
Comment (BurntSushi) [inline README.md]: Oh interesting. What is the difference between the `ripgrep` and `rust-ripgrep` packages?

**Comment by BurntSushi** [comment]
woohooo! Thanks @sylvestre!

**Comment by est31** [comment]
Nice, just thought today to check again on whether ripgrep will arrive in the next ubuntu or not. Thanks @sylvestre and @infinity0 ! Looking forward to the Ubuntu cosmic release.

**Comment by sylvestre** [comment]
rust-ripgrep is the name of the source package.  We are prefixing all rust packages to avoid conflicts in the naming (example rust-num). This is usually transparent for the user.
ripgrep is the name of the binary package.

File: README.md (hunk 1 above)
Comment (sdroege) [reply README.md]: The source package is called `rust-ripgrep`, the binary package is called `ripgrep`

**Pull request merged**